### PR TITLE
Manual SDK release v0.19.7

### PR DESCRIPTION
**Pull request by ngrayluna**
Docugen failing, under investigation thread here:
https://weightsandbiases.slack.com/archives/C0108SLEP6K/p1740150325002539

This PR was created by locally checking out docugen and using the SDK commit ID.

### CODE REVIEW

File: content/ref/cli/wandb-agent.md [hunk 1]
```diff
@@ -15,9 +15,9 @@ Run the W&B agent
 
 | **Option** | **Description** |
 | :--- | :--- |
-| -p, --project | The name of the project where W&B runs created from the   sweep are sent to. If the project is not specified, the run is sent to a project labeled 'Uncategorized'. |
-| -e, --entity | The username or team name where you want to send W&B   runs created by the sweep to. Ensure that the entity you specify already exists. If you don't specify an entity,   the run will be sent to your default entity, which is usually your username. |
-| --count | The max number of runs for this agent. |
+| `-p, --project` | The name of the project where W&B runs created from the   sweep are sent to. If the project is not specified, the run is sent to a project labeled 'Uncategorized'. |
```
Comment (mdlinville) [inline content/ref/cli/wandb-agent.md:18]: A nit that might be useful to follow up later if feasible -- these files could be more useful / easier to programmatically parse if the flags were separately marked up.

File: content/ref/cli/wandb-agent.md [hunk 2]
```diff
@@ -15,9 +15,9 @@ Run the W&B agent
...
+| `-e, --entity` | The username or team name where you want to send W&B   runs created by the sweep to. Ensure that the entity you specify already exists. If you don't specify an entity,   the run will be sent to your default entity, which is usually your username. |
```
Comment (mdlinville) [inline content/ref/cli/wandb-agent.md:19]: Nit for follow up later if feasible: It's a shame to see doubled / tripled spaces, like after `W&B` and after `entity,`. I see it in other files too -- if we are generating this programmatically perhaps the whitespace could be trimmed as part of that.

File: content/ref/cli/wandb-controller.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ Run the W&B local sweep controller
 
 | **Option** | **Description** |
 | :--- | :--- |
-| --verbose | Display verbose output |
+| `--verbose` | Display verbose output |
```
Comment (mdlinville) [inline content/ref/cli/wandb-controller.md:18]: I'm a little surprised on these `--verbose` flags we don't also support a short flag like `-v`?

File: content/ref/cli/wandb-init.md [hunk 4]
```diff
@@ -15,10 +15,10 @@ Configure a directory with Weights & Biases
...
 | **Option** | **Description** |
 | :--- | :--- |
-| -p, --project | The project to use. |
-| -e, --entity | The entity to scope the project to. |
-| --reset | Reset settings |
-| -m, --mode | Can be "online", "offline" or "disabled". Defaults to   online. |
+| `-p, --project` | The project to use. |
+| `-e, --entity` | The entity to scope the project to. |
+| `--reset` | Reset settings |
+| `-m, --mode` | Can be "online", "offline" or "disabled". Defaults to   online. |
```
[the remark this answers is not in the document]
Comment (mdlinville) [reply content/ref/cli/wandb-init.md:21]: To fix this, I suggest updating the source to use code font instead. I think I made that edit manually as part of migrating to Hug. Also nit about the tripled space, and about the non-Oxford comma.

File: content/ref/cli/wandb-launch.md [hunk 5]
```diff
@@ -15,23 +15,23 @@ Launch or queue a W&B Job. See https://wandb.me/launch
...
+| `-c, --config` | Path to JSON file (must end in '.json') or   JSON string which will be passed as a launch config. Dictation how the launched run will   be configured. |
+| `-v, --set-var` | Set template variable values for queues with   allow listing enabled, as key-value pairs e.g. `--set-var key1=value1 --set-var   key2=value2` |
+| `-q, --queue` | Name of run queue to push to. If none,   launches single run directly. If supplied without an argument (`--queue`), defaults to   queue 'default'. Else, if name supplied, specified run queue must exist under the   project and entity supplied. |
+| `--async` | Flag to run the job asynchronously. Defaults   to false, i.e. unless --async is set, wandb launch will wait for the job to finish. This   option is incompatible with --queue; asynchronous options when running with an   agent should be set on wandb launch-agent. |
+| `--resource-args` | Path to JSON file (must end in '.json') or   JSON string which will be passed as resource args to the compute resource. The exact   content which should be provided is different for each execution backend. See   documentation for layout of this file. |
```
Comment (mdlinville) [inline content/ref/cli/wandb-launch.md:32]: Nit: This should properly be code font instead of single quotes. Several extra spaces.

File: content/ref/cli/wandb-server/wandb-server-start.md [hunk 6]
```diff
@@ -15,9 +15,9 @@ Start a local W&B server
 
 | **Option** | **Description** |
 | :--- | :--- |
-| -p, --port | The host port to bind W&B server on |
-| -e, --env | Env vars to pass to wandb/local |
-| --daemon / --no-daemon | Run or don't run in daemon mode |
+| `-p, --port` | The host port to bind W&B server on |
+| `-e, --env` | Env vars to pass to wandb/local |
+| `--daemon / --no-daemon` | Run or don't run in daemon mode |
```
Comment (mdlinville) [inline content/ref/cli/wandb-server/wandb-server-start.md:20]: Odd. What's the default?

File: content/ref/python/agent.md [hunk 7]
```diff
@@ -2,7 +2,7 @@
 title: agent
 ---
 
-{{< cta-button githubLink=https://www.github.com/wandb/wandb/tree/v0.19.5/wandb/wandb_agent.py#L532-L576 >}}
+{{< cta-button githubLink=https://www.github.com/wandb/wandb/tree/2678738e59629208ad4770e3d36300a272147c05/wandb/wandb_agent.py#L532-L576 >}}
```
Comment (mdlinville) [inline content/ref/python/agent.md:5]: Are these correct? Shouldn't it be `0.19.7`?

**Comment by estellazx** [comment]
@ngrayluna should the title be `Manual SDK release v0.19.7` instead?

File: content/ref/python/public-api/api.md [hunk 8]
```diff
@@ -578,14 +636,49 @@ runs(
...
+- `$exists`
+- `$regex`
 
 #### Examples:
 
 Find runs in my_project where config.experiment_name has been set to "foo"
 
 ```
-api.runs(path="my_entity/my_project", filters={"config.experiment_name": "foo"})
+api.runs(
```
Comment (mdlinville) [inline content/ref/python/public-api/api.md:678]: Nit for followup later: Set the lexer as `python` in the source?

File: content/ref/python/watch.md [hunk 9]
```diff
@@ -22,12 +22,7 @@ extended to support arbitrary machine learning models in the future.
...
 | :--- | :--- |
-|  `models` | A single model or a sequence of models to be monitored. | 
-|`criterion `| The loss function being optimized (optional). |
-|`log `| Specifies whether to log `gradients`, `parameters`, or `all`. Set to `None` to disable logging. (default="gradients") |
-|`log_freq `|: How frequently to log gradients and parameters, expressed in batches. (default=1000)| 
-|`idx `| Index used when tracking multiple models with `wandb.watch`. (default=None)| 
-|`log_graph `| Whether to log the model's computational graph. (default=False) |
+|  models (Union[torch.nn.Module, Sequence[torch.nn.Module]]): A single model or a sequence of models to be monitored. criterion (Optional[torch.F]): The loss function being optimized (optional). log (Optional[Literal["gradients", "parameters", "all"]]): Specifies whether to log "gradients", "parameters", or "all". Set to None to disable logging. (default="gradients") log_freq (int): Frequency (in batches) to log gradients and parameters. (default=1000) idx (Optional[int]): Index used when tracking multiple models with `wandb.watch`. (default=None) log_graph (bool): Whether to log the model's computational graph. (default=False) |
```
Comment (mdlinville) [inline content/ref/python/watch.md:25]: Nit: These would all be fixed by using code font and removing the extra whitespace, in the source. Perhaps we need to socialize the sort of Markdown we would like to see used.

**Review by mdlinville — APPROVED** [review]
LGTM. Because this is new to me, I've left some feedback that I think might be worth following up later on in the source or in the transform from the source to this output.
